docker/hive/Dockerfile

```diff
@@ -0,0 +1,100 @@
+# trunk-ignore-all(terrascan/AC_DOCKER_0047)
+
+# CairoVM Chain
+FROM ghcr.io/dojoengine/dojo:v0.5.1 as katana
```

Comment:
do we need to use v0.6.0-alpha?

Reply:
[v0.6.0-alpha.0](https://github.com/dojoengine/dojo/releases/tag/v0.6.0-alpha.0)

Reply:
I prefer to stay on 0.5.1 for now and bump in another PR